src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonStringEnumConverter.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Text.Json.Serialization
     /// Converter to convert enums to and from strings.
     /// </summary>
     /// <remarks>
-    /// Reading is case insensitive, writing can be customized via a <see cref="JsonNamingPolicy" />.
+    /// Reading is case insensitive, writing can be customized via a <see cref="JsonKnownNamingPolicy" />.
```

Comment:
Shouldn't this be "or JsonNamingPolicy"? i.e. we aren't removing custom `JsonNamingPolicy` behavior.

Reply:
I didn't intend to change this at all - the new enum was originally called `JsonNamingPolicy` which caused naming and renaming issues. Will revert since everything boils down to an implementation of `JsonNamingPolicy`.